.github/workflows/ci-test.yml

```diff
@@ -18,19 +18,19 @@ on:
 
 jobs:
   build:
-    name: Auto-testing Framework / ${{ matrix.os }}
+    name: Auto-testing Framework / ${{ matrix.os }} / ${{ matrix.runtime }}
```

Comment:
let's not add cri-o-based test in the CI (no modification in ci-test.yml)

Reply:
Agreed @nam-jaehyun. I added them so that we could run tests on GH with cri-o for the changes in this PR... will undo.